wireapp/wire-ios-mono
# fix: transparent background on connect for iPad - WPB-12041

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-12041" title="WPB-12041" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-12041</a>  [iOS] Transparent background on the invite screen
  </td></table>
  <br />
 





### Issue

* fix background
* fix navigation flow on connect when accessing to contacts or asking for share permissions
* replace ZClientController.shared?.present with instance method `presentOverAll()` to avoid access to deprecated shared instance

### Testing

* login on iPad with personal account
* tap connect
* tap invite

* [x] check also flows on iPhone
